generators/client/files/state.js

```diff
@@ -0,0 +1,12 @@
+
+function load() {
+  try {
+    const element = document.getElementById('state');
+    return element ?
+      JSON.parse(element.textContent || element.innerText || '{}') : { };
+  } catch (e) {
+    return { };
+  }
+}
+
+export default load();
```

Comment:
Just have to be careful that the `state` script element is injected before this is loaded?